filecoin-proofs/src/api/mod.rs

```diff
@@ -318,6 +323,153 @@ where
     add_piece(source, target, piece_size, Default::default())
 }
 
+// Verifies if a DiskStore specified by a config is consistent.
+fn verify_store(config: &StoreConfig, arity: usize) -> bool {
```

Comment:
To make things a little more helpful, can we change this method to return `Result<()>`, and return things like `Missing store file: bla` or `Missing size` as errors if something doesn't match?